content/service/intro.md

```diff
@@ -0,0 +1,42 @@
++++
+# Homepage
+type = "blank"
+headless = true  # Homepage is headless, other widget pages are not.
+weight = 1
+title = ""
+
+[design]
+  columns = "1"
+
+[advanced]
+  css_class = "splash"
++++
+
+<div class="row">
+  <div class="col-12 col-lg-5">
+
+  <div><h1>JupyterHub in the cloud, managed by 2i2c</h1></div>
+
+  </div>
+  <div class="col-12 col-lg-7">
+
+{{<
+  figure src="/media/2i2c-hub-overview.png"
+>}}
+
+  </div>
+</div>
+
+<div class="subtext">
+
+We design, develop, and operate JupyterHubs in the cloud for your community in research and education. We handle the DevOps, you focus on your work.
```

Comment:
While not blocking this PR, I think it's really important to define what we mean by the word 'devops'. I think as it is used it probably means something else to you @choldgraf than what it means to me - I generally try to avoid it as it means too many things :) Can you expand a bit on what that word means to you?

Reply:
I'll open an issue to discuss so that we don't lose the thread in this PR!

Reply:
https://github.com/2i2c-org/team-compass/issues/256